sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -66,6 +66,15 @@ static void _rpl_trickle_send_dio(void *args)
           ipv6_addr_to_str(addr_str,&dodag->dodag_id, sizeof(addr_str)));
 }
 
+/* The lifetime of the default route should exceed the parent timeout interval
+ * by the time we allow the node to probe its parent */
+static uint16_t _dflt_route_lifetime(gnrc_rpl_dodag_t *dodag)
+{
+    return (dodag->default_lifetime * dodag->lifetime_unit) +
+           (GNRC_RPL_PARENT_TIMEOUT *
+            (GNRC_RPL_PARENT_PROBE_INTERVAL / MS_PER_SEC));
```

Comment:
If the function should return the time in ms, why divide by `MS_PER_SEC` here?

Reply:
It should not, it returns `s` but `GNRC_RPL_PARENT_PROBE_INTERVAL` is defined in `ms` :-)

Reply:
That's why I prefer to have the unit encoded in the name of the define :)